cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/NamespaceHttpHandlerTest.java

```diff
@@ -138,10 +139,17 @@ public void testCreateWithConfig() throws Exception {
     // create the custom namespace location first
     File customSpaceDir = tmpFolder.newFolder("data", "tmp", "myspace");
     // prepare - create namespace with config in its properties
-    String propertiesString = String.format("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":%s}",
-                                            NAME_FIELD, NAME, DESCRIPTION_FIELD, DESCRIPTION,
-                                            CONFIG_FIELD, "{ \"root.directory\": \"" + customSpaceDir.toString() +
-                                              "\", \"scheduler.queue.name\": \"testSchedulerQueueName\" }");
+
+
+    String propertiesString = GSON.toJson(ImmutableMap.of(NAME_FIELD, NAME, DESCRIPTION_FIELD, DESCRIPTION,
+                                                          CONFIG_FIELD,
+                                                          ImmutableMap.of(
```

Comment:
nit: can you break this to make it more readable? Perhaps make the config a separate variable?